tests/pytest_loganalyzer/pytest_loganalyzer.py

```diff
@@ -0,0 +1,181 @@
+import sys
+import logging
+import os
+import re
+
+from time import gmtime, strftime
+from os.path import join, split
+from os.path import normpath
+
+# Integration with legacy loganalyzer
+ANSIBLE_LOGANALYZER = normpath(join(split(__file__)[0], "../../ansible/roles/test/files/tools/loganalyzer/loganalyzer.py"))
+sys.path.append(split(ANSIBLE_LOGANALYZER)[0])
+from loganalyzer import LogAnalyzer as ansible_loganalizer
+
+COMMON_MATCH = "{}".format(join(split(ANSIBLE_LOGANALYZER)[0], "loganalyzer_common_match.txt"))
+COMMON_IGNORE = "{}".format(join(split(ANSIBLE_LOGANALYZER)[0], "loganalyzer_common_ignore.txt"))
+COMMON_EXPECT = "{}".format(join(split(ANSIBLE_LOGANALYZER)[0], "loganalyzer_common_expect.txt"))
+SYSLOG_TEMP_FOLDER = "/tmp/pytest-run/syslog"
+
+
+class PytestLogAnalyzer:

```

Comment:
In general LogAnalyzer is test framework agnostic, no need for prefix 'Pytest'

Reply:
We already have loganalyzer.py with class name LogAnalyzer.
It is also bad to duplicate module and class name in the same time.
Thoughts?

Reply:
Suggest to rename original loganalyzer.py and class in there. Let's here thoughts tomorrow from the team about name.

Reply:
Renamed only class name to AnsibleLogAnalyzer